grammars/gfm.cson

```diff
@@ -50,9 +50,12 @@
     'name': 'markup.strike.gfm'
   }
   {
-    'begin': '^#{6}\\s*'
+    'begin': '^(#{6}\\s*)'
```

Comment:
Are you sure you want to include the whitespace (`\s*`) in the capture group?

Would then make it tricky if you wanted to style the `#` to be underlined for example because you would also have the whitespace after `#` underlined as well.